web-app/src/components/tournament/creator/teamBuilder.vue

```diff
@@ -199,10 +201,10 @@ export default class TeamBuilder extends Vue {
   private validationName() {
     if (this.teamName.length <= 3) {
       this.isErrorInputTeamName = true;
-      this.teamNameErrorMessage = this.$t('tournament.tooShortTeamNameError');
+      this.teamNameErrorMessage = this.$t('tournament.team.error.tooShortName');
     } else if (this.teamName.length > 40) {
       this.isErrorInputTeamName = true;
-      this.teamNameErrorMessage = this.$t('tournament.tooLongTeamNameError');
+      this.teamNameErrorMessage = this.$t('tournament.team.error.tooLongName');
     } else {
```

Comment:
this code is required? When we have `maxlength` in component this will never happen. Right? i'm not sure. So this lines and `tooLongName` in locales are useless

Reply:
true, it's useless after adding maxlength in input